### PR TITLE
#N/A: Refactor for better readability

### PR DESCRIPTION
**Pull request by divykj**

### CODE REVIEW

**Comment by divykj** [comment]
Hey @scorphus, could you or someone else review this, and give feedback?
IDK if this was needed or not, but I tried to make the test more readable.

File: tests/rules/test_fix_file.py [hunk 1]
```diff
@@ -2,62 +2,54 @@
 
 import pytest
 import os
+from collections import namedtuple
 from thefuck.rules.fix_file import match, get_new_command
 from thefuck.types import Command
 
+FileFixTest = namedtuple('FileFixTest', ['script', 'file', 'line', 'col', 'output'])
```
Comment (scorphus) [inline tests/rules/test_fix_file.py:9]: Nit:
```suggestion
FixFileTest = namedtuple('FixFileTest', ['script', 'file', 'line', 'col', 'output'])
```

Comment (divykj) [reply tests/rules/test_fix_file.py]: Done.

**Comment by divykj** [comment]
@nvbn I resolved the conflicts and now there are no warnings.

**Comment by nvbn** [comment]
Thanks!

**Pull request merged**